### ISSUE TITLE
phoenix tests failing due to publishing hanging while using DataCite

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#5409]
At standup I mentioned that phoenix tests are failing right now which always makes me nervous if we're close to a release and 4.10 is imminent. The current commit is 0b625dd but the new failures (new since the single "testPrivateUrl" failure in #5400 I mean) appear to be due to publishing hanging while using DataCite rather than anything in the code.

Pull request #5399 already contains a line to switch phoenix over to the new "FAKE" PID provider so it can be a fix for this issue:

![screen shot 2018-12-18 at 11 39 34 am](https://user-images.githubusercontent.com/21006/50168413-62b8e300-02b9-11e9-9823-db30fd9c57d4.png)

For anyone unfamiliar with the phoenix server, please see http://guides.dataverse.org/en/4.9.4/developers/testing.html#the-phoenix-server

To give a sense of the failures, here are screenshots from

https://build.hmdc.harvard.edu:8443/job/phoenix.dataverse.org-apitest-develop/311/

![screen shot 2018-12-18 at 11 34 19 am](https://user-images.githubusercontent.com/21006/50168296-11105880-02b9-11e9-976c-9e8f41cafa7b.png)

![screen shot 2018-12-18 at 11 34 27 am](https://user-images.githubusercontent.com/21006/50168294-1077c200-02b9-11e9-9d37-5d1f9fbec46b.png)

Part of the challenge for me is that when I look at https://status.datacite.org I don't know which of the currently 4 "Major Outage" indicators I should expect to affect the phoenix server, if any. Here's a screenshot:

![screen shot 2018-12-18 at 11 40 58 am](https://user-images.githubusercontent.com/21006/50168522-a14e9d80-02b9-11e9-96d7-0f19abc7edf8.png)

I'd like to point out that while we **could** switch phoenix back to EZID, we switched it away from EZID for a reason in #4725. We were being told phoenix was DOS'ing EZID. For all I know, the phoenix IP address is still blocked by EZID.

The bottom line is that merging pull request #5399 should get the tests passing again on phoenix.

**Comment by pdurbin** [comment]
Now that pull request #5399 (and #5407) have been merged, all API tests are passing on phoenix again (build 312 in the screenshot below).

![screen shot 2018-12-18 at 4 39 20 pm](https://user-images.githubusercontent.com/21006/50184551-6d886d80-02e3-11e9-9ed3-9f6ef90806ce.png)
